### PR TITLE
Cucumber-Ruby-Assignment

### PR DESCRIPTION
**Pull request by madhumitha-bs**
Latest run results: 
bstack automate result
local result

Latest Build from Automate: https://automate.browserstack.com/dashboard/v2/public-build/cGRKSzVZbGJVWXlzMGpXZExPY1hjK2lOTWZlVVZFRFBROHkyQ0VMdlV6MFV6N3lDUlovSnFnQndiTWorU1ZFYkI3T25VTGFJY21sNXJseHArenZRZ3c9PS0tREgwWmxiZU1sZitEM1o4dndvVGFRZz09--d1be411edc0d959d18d0df657cc157c5a68d4f2d

### CODE REVIEW

File: cucumber-ruby assignment/features/feature_files/example.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    When User search for "<product>"
+    Then User should see search results for "<product>"
+  
+  Examples:
+      | product       |
+      | laptop        |
+      | mobile phone  |
+      | headphones    |
+      | smart watch   |
+      | tablet        |
```
Comment (Sameer-Dhole) [inline cucumber-ruby assignment/features/feature_files/example.feature]: Add Newline

File: cucumber-ruby assignment/Rakefile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+require 'rake'
+require 'cucumber/rake/task'
+
+Cucumber::Rake::Task.new(:features) do |t|
+  t.cucumber_opts = ['features', '--format', 'pretty']
+end
+
+task default: :features
```
Comment (Sameer-Dhole) [inline cucumber-ruby assignment/Rakefile]: add newline

File: cucumber-ruby assignment/Gemfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Gemfile
+source 'https://rubygems.org'
+
+gem 'cucumber'
+gem 'selenium-webdriver'
+gem 'rake'
+gem 'rspec'
+gem 'capybara'
+gem 'page-object'
+gem 'browserstack-local'
```
Comment (Sameer-Dhole) [inline cucumber-ruby assignment/Gemfile]: add new line

File: cucumber-ruby assignment/features/support/env.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+require 'selenium-webdriver'
+require 'selenium-webdriver'
+require 'cucumber'
+require 'rspec'
+
+USER_NAME = ENV['BROWSERSTACK_USERNAME']
+ACCESS_KEY = ENV['BROWSERSTACK_ACCESS_KEY']
+BUILD_NAME = ENV['BROWSERSTACK_BUILD_NAME']
+
+$page_url = 'https://www.flipkart.com'
```
Comment (Sameer-Dhole) [inline cucumber-ruby assignment/features/support/env.rb]: add newline

**Comment by Sameer-Dhole** [comment]
One more feedback , make the test name dynamic instead of "Ruby thread"

**Comment by madhumitha-bs** [comment]
> One more feedback , make the test name dynamic instead of "Ruby thread"

Hi @Sameer-Dhole, I have committed and pushed the changes. Results from latest build: https://automate.browserstack.com/dashboard/v2/public-build/SCtqWFdIakZPU0hpUUszaGlGTlJZUG5Rc3BFQjNzL0w2VjYrSzlmUEhrOEZzRys2Qys1N0lqbTU1M2RFQlhoK0JSVm9GOWVaVHozWm81M0pYZjllcGc9PS0tYklNOTBBaHI3ZHd2a21DUGpEOGl1UT09--626ece3262a4ddc139fa2124c30552c441458f13

**Comment by Sameer-Dhole** [comment]
Looks Good

**Pull request merged**